`area_kinki_trafficinfo`: `ValidationError: 2 validation errors for TrainInfo`

Ref: #51

`area_kinki_trafficinfo` で `TrainInfo` の `from_` を `Optional` にする必要がある。

---

https://github.com/unyacat/westjr/actions/runs/5860277526/job/15888001740?pr=59

```
westjr/api.py:46: ValidationError
=========================== short test summary info ============================
FAILED tests/test_example.py::test_get_traffic_info - pydantic_core._pydantic_core.ValidationError: 2 validation errors for TrainInfo
lines.kinokuni.section.from
  Input should be a valid string [type=string_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.1/v/string_type
lines.wakayama1.section.from
  Input should be a valid string [type=string_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.1/v/string_type
```

--- a/westjr/response_types/train_info.py
+++ b/westjr/response_types/train_info.py
@@ -5,7 +5,7 @@ from pydantic import BaseModel, Field
 
 
 class Section(BaseModel):
-    from_: str = Field(..., alias="from")
+    from_: Optional[str] = Field(..., alias="from")
     to: Optional[str] = None
 
 

